apps/meteor/tests/e2e/admin-users.spec.ts

```diff
@@ -41,48 +46,148 @@ test.describe('Admin > Users', () => {
-				await expect(page.getByRole('link', { name: username })).toBeVisible();
+				await admin.tabs.users.tabAll.click();
+				await expect(admin.getUserRowByUsername(username)).toBeVisible();
 			});
 
 			await test.step('is visible in the Pending tab', async () => {
-				await page.getByRole('tab', { name: 'Pending' }).click();
-				await expect(page.getByRole('link', { name: username })).toBeVisible();
+				await admin.tabs.users.tabPending.click();
+				await expect(admin.getUserRowByUsername(username)).toBeVisible();
 			});
 
 			await test.step('is not visible in the Active tab', async () => {
-				await page.getByRole('tab', { name: 'Active' }).click();
-				await expect(page.getByRole('link', { name: username })).not.toBeVisible();
+				await admin.tabs.users.tabActive.click();
+				await expect(admin.getUserRowByUsername(username)).not.toBeVisible();
 			});
 
 			await test.step('is not visible in the Deactivated tab', async () => {
-				await page.getByRole('tab', { name: 'Deactivated' }).click();
-				await expect(page.getByRole('link', { name: username })).not.toBeVisible();
+				await admin.tabs.users.tabDeactivated.click();
+				await expect(admin.getUserRowByUsername(username)).not.toBeVisible();
 			});
 
 			await test.step('moves from Pending to Deactivated tab', async () => {
 				await page.getByRole('tab', { name: 'Pending' }).click();
-				await page.getByRole('button', { name: 'More actions' }).click();
-				await page.getByRole('menuitem', { name: 'Deactivate' }).click();
-				await expect(page.getByRole('link', { name: username })).not.toBeVisible();
-
-				await page.getByRole('tab', { name: 'Deactivated' }).click();
-				await expect(page.getByRole('link', { name: username })).toBeVisible();
+				await admin.tabs.users.btnMoreActionsMenu.click();
+				await admin.tabs.users.menuItemDeactivated.click();
+				await expect(admin.getUserRowByUsername(username)).not.toBeVisible();
+				await admin.tabs.users.tabDeactivated.click();
+				await expect(admin.getUserRowByUsername(username)).toBeVisible();
 			});
 
 			await test.step('moves from Deactivated to Pending tab', async () => {
-				await page.getByRole('tab', { name: 'Deactivated' }).click();
-				await page.getByRole('button', { name: 'More actions' }).click();
-				await page.getByRole('menuitem', { name: 'Activate' }).click();
-				await expect(page.getByRole('link', { name: username })).not.toBeVisible();
-
-				await page.getByRole('tab', { name: 'Pending' }).click();
-				await expect(page.getByRole('link', { name: username })).toBeVisible();
+				await admin.tabs.users.tabDeactivated.click();
+				await admin.tabs.users.btnMoreActionsMenu.click();
+				await admin.tabs.users.menuItemActivate.click();
+				await expect(admin.getUserRowByUsername(username)).not.toBeVisible();
+				await admin.tabs.users.tabPending.click();
+				await expect(admin.getUserRowByUsername(username)).toBeVisible();
 			});
 		},
 	);
 });
+
+test.describe('Login', () => {
```

Comment:
the describe below is dependent on this test, right? If it runs in parallel, it could be an issue